codegen/templates/http_client.tmpl

```diff
@@ -123,19 +124,32 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 	if deps.Default.Config.ContainsKey("clients.{{$clientID}}.circuitBreakerDisabled") {
 		circuitBreakerDisabled = deps.Default.Config.MustGetBoolean("clients.{{$clientID}}.circuitBreakerDisabled")
 	}
+
+	//get mapping of client method and it's timeout
+	//if mapping is not provided, use client's timeout for all methods
```

Comment:
Might be too late but `method` **isn't the right nomenclature**. We should be pushing for `procedure` instead to stay in parity with a general router layer whose sole purpose is to dispatch procedures calls to remote outbounds (similar to Muttley)

Reply:
I've no problem with that. My only concern is that we should be calling it as `procedure` instead of `method`

Reply:
So I have the correct understanding. My only point was that we should start calling it `procedures` instead of methods as a platform. E.g `ep level procedure timeouts` instead of `ep level method timeouts`

But this is just a nit.